README.rst

```diff
@@ -1681,6 +1681,308 @@ Note: these six options ``--unencrypted-suffix``, ``--encrypted-suffix``, ``--en
+
+* ``azure_keyvault`` (list of objects): list of Azure Key Vault resource identifiers.
+  Every object must have the following keys:
+
+  * ``vaultUrl`` (string): the vault URL.
+
+  * ``key`` (string): the key name.
+
+  * ``version`` (string, can be empty): the version of the key to use.
+    If empty, the latest key will be used on encryption.
+
+  Example:
+
+  .. code:: yaml
+
+    - vaultUrl: https://vault.url
+      key: key-name
+      version: key-version
+    - vaultUrl: https://vault.url
+      key: key-name
+      version: ""
+
+* ``hc_vault`` (list of strings): list of HashiCorp Vault transit URIs.
+
+* ``age`` (list of strings): list of Age public keys.
+
+* ``pgp`` (list of strings): list of PGP/GPG key fingerprints.
+
+* ``merge``: a list of key group objects.
+  These will be merged (by concatenating the keys of the same type) into this key group.
+  This key is only there to allow concatenation of key groups using YAML anchors, aliases, and overrides.
+
+Settings
+********
+
+The following keys configure encryption settings:
+
+* ``shamir_threshold`` (integer, default ``0``): Must be ``0`` (disabled) or an integer greater or equal to 2.
+  Determines the number of key groups from whose one key must be present each to decrypt the file's key.
+
+* ``mac_only_encrypted`` (boolean, default ``false``): If set to ``true``, only encrypted strings will count towards the file's MAC.
+  If set to ``false``, also unencrypted values will be part of the MAC computation.
+
+The following keys configure which values in a file are encrypted.
+Note that at most one of these keys can be used.
+
+* ``unencrypted_suffix`` (string): A value is encrypted if its key **does not** end with this suffix.
+  All other values are **encrypted**.
+  Comments are always encrypted.
+
+* ``encrypted_suffix`` (string): A value is encrypted if its key **does** end with this suffix.
+  All other values are **not encrypted**.
+  Comments are always encrypted.
+
+* ``unencrypted_regex`` (string): A value is encrypted if its key **does not match** this regular expression.
+  All other values are **encrypted**.
+  Comments are always encrypted.
+
+* ``encrypted_regex`` (string): A value is encrypted if its key **matches** this regular expression.
+  All other values are **not encrypted**.
+  Comments are always encrypted.
+
+* ``unencrypted_comment_regex`` (string): A value or comment is not encrypted if it has a preceding comment (or a trailing comment on the same line) matching this regular expression.
+  All other values and comments are encrypted.
+
+* ``encrypted_comment_regex`` (string): A value or comment is encrypted if it has a preceding comment (or a trailing comment on the same line) matching this regular expression.
+  All other comments and values are not encrypted.
+  The matching comment itself is not encrypted.
+
+Destination rule object
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Not yet documented.
+
+.. TODO!
```

Comment:
Lets remove the TODO line as we already have a callout for the section of the document that is lacking content

Reply:
It can stay. RST is indeed "fun"